### DISCUSSION TITLE
Webmention support?

### DISCUSSION THREAD

**Discussion by nigelsim** [importantimport/hatsu#81] [General]
Hi, fantastic project. I'm curious if you are intending on adding support for other IndieWeb protocols, specifically Webmentions? 

Given the use case of a static site, we'd probably want to run as few supporting services as possible, so having Webmentions, and maybe one day comments a la [Remark42](https://remark42.com/) then basically have a full feature blog system.

If not, how extendable do you think this project would be so that someone could add these features without it becoming a hard fork? I've never written Rust so I can't judge that for myself.

Thanks

**Comment by kwaa** [comment]
Hatsu plans to support Webmention, but is still in preparation because it is relatively complicated.

**Reply by kwaa** [reply]
Currently, it is blocked by #7
